impeller/docs/renderdoc_frame_capture.md

```diff
@@ -1,16 +1,15 @@
 # Frame Capture with RenderDoc
 
-[RenderDoc](https://renderdoc.org/) is a graphics debugger that can be used to capture frames. With Impeller starting to support gles and Vulkan backends, RenderDoc can provide insights into the application's frames.
+[RenderDoc](https://renderdoc.org/) is a graphics debugger that can be used to capture frames. With Impeller starting to support OpenGL-ES and Vulkan backends, RenderDoc can provide insights into the application's frames.
 
 1. First step is to set up RenderDoc. Follow the instructions [here](https://renderdoc.org/docs/getting_started/quick_start.html). For the purposes of this guide it is assumed that you are able to get RenderDoc running.
 2. The next step would be to run the application you wish the capture the frames of. Typically these would be one of the playground tests. To build these, do:
 
 ```bash
 # In your $ENGINE_SRC folder, do:
 
-./flutter/tools/gn --unopt --no-lto --enable-impeller-playground\
-   --target-dir host_debug_impeller_vulkan
-autoninja -C out/host_debug_impeller_vulkan/
+./flutter/tools/gn --unopt --no-lto --enable-impeller-vulkan
+autoninja -C out/host_debug_unopt/
```

Comment:
nit: I might just s/autoninja/ninja given at least this week autoninja vendored by depot_tools doesn't work in Flutter.